angular.json

```diff
@@ -23,7 +23,14 @@
               "node_modules/jquery/dist/jquery.js",
               "node_modules/jquery-ui-dist/jquery-ui.js",
               "src/assets/vmware-wmks/js/wmks.min.js"
-            ]
+            ],
+            "aot": false,
```

Comment:
Why is aot false here?

Reply:
No idea, I can change it -- we only use production mode anyway where aot is true, but yeah I'm not sure why the migration put that, besides maybe that's the default if it didn't exist?